tests/cephfs/snapshot_clone/cephfs_snap_utils.py

```diff
@@ -221,6 +230,11 @@ def activate_snap_schedule(self, client, path, **kw_args):
         for eg., snap_util.activate_snap_schedule(client,activate_path,sched_val=sched_val)
         """
         cmd = f"ceph fs snap-schedule activate {path}"
+        if kw_args.get("subvol_name"):
+            sv_name = kw_args["subvol_name"]
+            cmd = f"ceph fs snap-schedule activate / --subvol {sv_name}"
```

Comment:
Hi Suma
should it be like this `ceph fs snap-schedule activate /` --> `ceph fs snap-schedule activate {path}`
or can you keep appending the cmd as in if the kwargs are present as what has been done for group_name
 cmd = f"ceph fs snap-schedule activate / --subvol {sv_name}" -->  cmd += f" --subvol {sv_name}"

Reply:
If subvol option is not used, path will be what user has set which is getpath value of subvol and append '/..'.
If subvol option is used path has to be '/' or '-', as per cephfs documentation. the complete path is not necessary to be given if subvol option is used, hence i did not cover that approach.